assets/xml/objects/object_bat.xml

```diff
@@ -1,17 +1,20 @@
 ﻿<Root>
+    <!-- Displaylists for ovl_En_Bat (Bad Bats) -->
     <File Name="object_bat" Segment="6">
```

Comment:
Since there are two textures at the bottom, I don't think this comment should apply to the whole file?

Reply:
Yeah, it's tricky to know how to describe this one since there's not really a "model" *per se*, just a lot of displaylists and the vtx/textures they need.